src/libcprover-rust/include/c_api.h

```diff
@@ -3,6 +3,7 @@
 #pragma once
 
 #include <memory>
+#include <stdexcept>
```

Comment:
Why has this file being touched by this PR?
It shouldn't be changed.

Reply:
removed, sorry for cluttering